src/components/sections/Hero/Hero.tsx

```
@@ -11,50 +12,101 @@ export const Hero = (): React.ReactElement => {
+                </div>
+              </div>
+              <div className={styles.mockupStats}>
+                <div className={styles.mockupStatCard}>
+                  <div className={styles.mockupStatIcon}>
+                    <Icon name="fa-clock" size="sm" />
+                  </div>
+                  <span className={styles.mockupStatLabel}>PRODUCTIVITY</span>
+                  <span className={styles.mockupStatValue}>+120%</span>
+                  <span className={styles.mockupStatDesc}>Focus increase reported by users</span>
+                </div>
+                <button className={styles.mockupShareBtn}>Share</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)